daft/dataframe/dataframe.py

```diff
@@ -889,6 +891,56 @@ def join(
         )
         return DataFrame(join_op)
 
+    @DataframePublicAPI
+    def drop_na(self, *cols: ColumnInputType):
```

Comment:
Ah - I see where the source of confusion here might be!!

We were intending to have `na` mean "both null and nan", and have separate methods:

1. `drop_na` (drops both nan float and null values)
2. `drop_nan` (drops nan float values)
3. `drop_null` (drops null values)

Essentially (1) `drop_na` performs both (2) and (3).

This might be confusing though - let me know what you think?

I think a good compromise might be to just rename this function to `.drop_nan` and call it a day :) that way users won't have to understand what we mean by "na" which could be very confusing... As you've just experienced 😛